src/mock.rs

```diff
@@ -1,37 +1,37 @@
 // FIXME: re-implement tests with `async/await`
 /*
-#[cfg(feature = "runtime")]
+#[cfg(runtime)]
```

Comment:
Doing some more digging, this feature doesn't exist in the cargo.toml. Should it be removed from the source or added to the features list?